File: cpp/C++ #38-46/05.cpp
#include <iostream>
using namespace std ;
int main(){

    int vals[] = {100, 200, 600, 200, 100};
    // int vals[] = {100, 200, 200, 100};
    // int vals[] = {100, 300, 600, 200, 100};

    int start = 0 ;
    int end = sizeof(vals) / sizeof(int) -1;

    if (vals[start] == vals[end]){
        if (vals[++start] == vals[--end])
            cout << "Array Is Palindrome\n" ;

        else 
            cout <<"Array Is Not Palindrome\n";
    }
    else 
        cout <<"Array Is Not Palindrome\n";
    
    return 0;
}

File: PHP/PHP #37-42/05.php
<?php
$start = 0;
$mix = [1, 2, 3, "A", "B", "C", 4];

echo "\n\n" ;

for  ($start+=1; $start < sizeof($mix) ; $start ++){

    echo (gettype($mix[$start]) ==="integer") ? $mix[$start] . "\n" : null ;
}

?>